src/App.tsx

```diff
@@ -21,6 +42,7 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
```

Comment:
```suggestion
            className="form-control"
            value={perPage}
```
provide value to select. it is should fix one of failed tests.